src/base/DelegateInputLabelMixin.js

```diff
@@ -31,18 +32,60 @@ export default function DelegateInputLabelMixin(Base) {
       this[setState]({ ariaLabel });
     }
 
+    // Forward ARIA labelledby as an aria-label to the input element.
+    get ariaLabelledby() {
+      return this[state].ariaLabelledby;
+    }
+    set ariaLabelledby(ariaLabelledby) {
+      if (this.getRootNode() !== null) {
```

Comment:
Sometimes people create components in memory and then add them to a document. In that scenario, when they set the `ariaLabelledby` property, it will trigger an immediate search for the associated element — which will fail because the component isn't in the document yet.

Recommend deferring this search to after the component is in the document. The easiest way to do that is to put the code in `render` or `rendered`. Since this doc needs to set state, I'd go with `rendered`.